script/build_ebpf_xdp.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright 2025 Intel Corporation
+
+set -e
+
+script_name=$(basename "${BASH_SOURCE[0]}")
+script_path=$(readlink -qe "${BASH_SOURCE[0]}")
+script_folder=${script_path/$script_name/}
+cd "${script_folder}"
+
+(return 0 2>/dev/null) && sourced=1 || sourced=0
+
+if [ "$sourced" -eq 0 ]; then
+	cd "${script_folder}"
+
+	if [ ! -d "xdp-tools" ]; then
+		echo "Clone XDP source code"
+		git clone --recurse-submodules https://github.com/xdp-project/xdp-tools.git
```

Comment:
Use instead:

```shell
. "./common.sh"

git_download_strip_unpack "xdp-project/xdp-tools"  "main" "${script_folder}/xdp-tools"
git_download_strip_unpack ""libbpf/libbpf""  "main" "${script_folder}/xdp-tools/lib/libbpf"
```

Reply:
this script is set to emulate the documentation,
we don't really want users to source the common i think it's fine this way :?